src/worker.ts

```diff
@@ -1,12 +1,17 @@
 import { CronJob } from 'cron'
+import { PrismaMessageRepository } from './repositories/prisma/prisma-message-repository';
+import { SendMailUseCase } from './use-cases/send-mail-use-case';
 
 const job = new CronJob(
   '0 8/24 * * *',
-  () => {
-    console.log(new Date().toString())
+  async () => {
+    const prismaMessageRepository = new PrismaMessageRepository();
+    const useCase = new SendMailUseCase(prismaMessageRepository)
+
+    await useCase.execute({ targetDate: new Date })
```

Comment:
Ficou faltando os parênteses no construtor de data.

```suggestion
    await useCase.execute({ targetDate: new Date() })
```